src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ChannelPricing.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%sylius.model.channel_pricing.class%" shortName="ChannelPricing">
```

Comment:
If I think correctly, with this configuration you are exposing the every default endpoints (get, post, put, delete) for channel pricing. You should define only these operations that you need.